### PR TITLE
First pass PIF merging with (scrubbed public PIF-based) tests.

### PR DESCRIPTION
**Pull request by sparadiso**
@kjaym

### CODE REVIEW

**Comment by sparadiso** [comment]
Rebuild me!

**Comment by sparadiso** [comment]
Rebuild me!

File: pom.xml [hunk 1]
```diff
@@ -159,6 +190,34 @@
         </plugins>
     </build>
 
+    <repositories>
```
Comment (kjaym) [inline pom.xml]: I'm so sorry @sparadiso, but I think that you know what needs to happen.

Comment (sparadiso) [reply pom.xml]: Whatever could you possibly be referring to?

**Comment by maxhutch** [comment]
LGTM!  Ready to merge?

**Comment by sparadiso** [comment]
@kjaym @maxhutch ready 😭

**Comment by sparadiso** [comment]
@kjaym we should update the style settings in https://github.com/CitrineInformatics/citrine-style/tree/master/IntelliJ to be compatible with the Java checkstyle settings.

Standardizing on a style settings file made our (max and my) lives much easier when working on nucleus-ml together.

**Comment by sparadiso** [comment]
@kjaym afaik this is now ready

**Pull request merged**